src/SPH/SPHHydroBase.hh

```diff
@@ -217,6 +217,17 @@ public:
 
 protected:
   //---------------------------  Protected Interface ---------------------------//
+  // The templated derivative method, allowing us to use different types of kernels
+  template<typename KernelType>
+  void evaluateDerivativesImpl(const Scalar time,
+                               const Scalar dt,
+                               const DataBase<Dimension>& dataBase,
+                               const State<Dimension>& state,
+                               StateDerivatives<Dimension>& derivatives,
+                               const KernelType& W,
+                               const KernelType& WQ,
+                               const TableKernel<Dimension>& WT) const;
+
```

Comment:
So pretty much just moved SPHEvaluateDerivatives into SPHHydroBase, correct? Or are there some other more detailed changes that are harder to see with the git diffs?

Reply:
Yeah, this is mostly just collapsing the evaluateDerivatives back into the .cc file since we have so many SPH variants now.  I split this out years ago to help with parallel building, but the new Cmake system parallelizes the build quite well so it's no longer necessary.

There are a couple of other minor changes in the implementation due to changes in the kernel interface I implemented getting the SphericalKernel done.  Nothing substantive.